### PR TITLE
Run flux e2e test group in GHA's local_e2e_tests job

### PR DESCRIPTION
**Pull request by gfichtenholt**
Description of the change

It seems that at some point, a new flux test group has been added to the /script/e2e_test.sh file, but we have forgotten to add it to the GHA's local_e2e_tests job, so this test group is not being run in the GHA workflow.

I have also took advantage of this PR to simplify the local_e2e_tests GHA job by setting a timeout of 6 minutes for every tests group, instead of having a default of 4 and setting it to 6 for the operator's group to avoid flaky tests.
Benefits

    Run all the e2e tests from GHA.

Possible drawbacks

N/A

### CODE REVIEW

File: .github/workflows/kubeapps-general.yaml [hunk 1]
```diff
@@ -609,10 +602,9 @@ jobs:
       - push_images
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
       - uses: ./.github/actions/srp-source-provenance
         with:
-          SRP_CLI_VERSION: ${{env.SRP_CLI_VERSION}}
+          SRP_CLI_VERSION: ${SRP_CLI_VERSION}
```
Comment (absoludity) [inline .github/workflows/kubeapps-general.yaml:607]: I think this is an unintentional change - but should go away if you merge main? (I saw @beni0888 fix this in main earlier today - adding the line that your PR is reverting here by accident).

**Review by absoludity — APPROVED** [review]
Thanks Greg.

File: .github/workflows/kubeapps-general.yaml (hunk 1 above)
Comment (gfichtenholt) [reply .github/workflows/kubeapps-general.yaml:607]: yes, don't worry about it. I am in a testing phase

**Comment by gfichtenholt** [comment]
latest debug info:
flux's source-controller deployment fails after 5 retries
```
+ kubectl describe deployment --namespace flux-system source-controller
Name:               source-controller
Namespace:          flux-system
CreationTimestamp:  Thu, 17 Nov 2022 10:21:38 +0000
Labels:             app.kubernetes.io/component=source-controller
                    app.kubernetes.io/instance=flux-system
                    app.kubernetes.io/part-of=flux
                    control-plane=controller
Annotations:        deployment.kubernetes.io/revision: 1
Selector:           app=source-controller
Replicas:           1 desired | 1 updated | 1 total | 0 available | 1 unavailable
StrategyType:       Recreate
MinReadySeconds:    0
Pod Template:
  Labels:           app=source-controller
  Annotations:      prometheus.io/port: 8080
                    prometheus.io/scrape: true
  Service Account:  source-controller
  Containers:
   manager:
    Image:       ghcr.io/fluxcd/source-controller:v0.31.0
    Ports:       9090/TCP, 8080/TCP, 9440/TCP
    Host Ports:  0/TCP, 0/TCP, 0/TCP
    Args:
      --events-addr=http://notification-controller.flux-system.svc.cluster.local./
      --watch-all-namespaces
      --log-level=info
      --log-encoding=json
      --enable-leader-election
      --storage-path=/data
      --storage-adv-addr=source-controller.$(RUNTIME_NAMESPACE).svc.cluster.local.
    Limits:
      cpu:     1
      memory:  1Gi
    Requests:
      cpu:      50m
      memory:   64Mi
    Liveness:   http-get http://:healthz/healthz delay=0s timeout=1s period=10s #success=1 #failure=3
    Readiness:  http-get http://:http/ delay=0s timeout=1s period=10s #success=1 #failure=3
    Environment:
      RUNTIME_NAMESPACE:   (v1:metadata.namespace)
      TUF_ROOT:           /tmp/.sigstore
    Mounts:
      /data from data (rw)
      /tmp from tmp (rw)
  Volumes:
   data:
    Type:       EmptyDir (a temporary directory that shares a pod's lifetime)
    Medium:     
    SizeLimit:  <unset>
   tmp:
    Type:       EmptyDir (a temporary directory that shares a pod's lifetime)
    Medium:     
    SizeLimit:  <unset>
Conditions:
  Type           Status  Reason
  ----           ------  ------
  Available      False   MinimumReplicasUnavailable
  Progressing    False   ProgressDeadlineExceeded
OldReplicaSets:  <none>
NewReplicaSet:   source-controller-57ff555bd (1/1 replicas created)
Events:
  Type    Reason             Age   From                   Message
  ----    ------             ----  ----                   -------
  Normal  ScalingReplicaSet  10m   deployment-controller  Scaled up replica set source-controller-57ff555bd to 1
+ kubectl get pods --namespace flux-system
NAME                                          READY   STATUS    RESTARTS   AGE
helm-controller-7dff4d4dcd-nl6ps              1/1     Running   0          10m
image-automation-controller-c65788ff4-l5m6f   1/1     Running   0          10m
image-reflector-controller-7f4b74ff85-zd69b   1/1     Running   0          10m
kustomize-controller-677586bf9-j82ct          1/1     Running   0          10m
notification-controller-6786d46b49-9hpcf      1/1     Running   0          10m
source-controller-57ff555bd-8brds             0/1     Pending   0          10m
```

Will look into why MinimumReplicasUnavailable might happen next

**Comment by beni0888** [comment]
Great work @gfichtenholt !! 👏🏻 👏🏻

**Pull request merged**